lessons/cryptography/cryptosystem-types/cryptosystems-types.md

```diff
@@ -0,0 +1,90 @@
+# [Cryptography] Cryptosystem Types
+*Learn about modern cryptosystems that are in use today.*
+
+![banner alt text](.rsrc/banner.png)
+
+# Objectives
+- Symmetric vs asymmetric cryptography (public key)
+- Symmetric (AES)
+- Asymmetric (RSA, diffie-hellman, elliptic curve)
+
+# Introduction
+In a previous lesson, we talked about several primitive cryptosystems. Now, we will cover modern systems that are in use today. Modern cryptography is often categorized as symmetric or asymmetric cryptography. Symmetric cryptography requires both the sender and receiver of an encrypted message to have a shared private key. In asymmetric cryptography, the sender and receiver each have a public and a private key, and only need to share their public keys in order to exchange encrypted messages. 
+
+In this lesson, we will cover the basics of a few types of symmetric (AES) and asymmetric (RSA, Diffie-Hellman, Elliptic Curve) cryptosystems. 
+
+# Lesson
+## Symmetric Cryptosystems
+
+In a symmetric cryptosystem, both the sender and receiver of an encrypted message needs to know the private key. Prior to sending and receive any secure messages, both parties must find a way to securely decide upon a private key. In the example below, Alice is sending an crypted message to Bob. 
+
+![Sending a Message with Symmetric Cryptography](.rsrc/symmetric-crypto.png)
+
+Alice encrypts a plaintext message with the shared private key, resulting in a ciphertext message. This encrypted ciphertext message is sent to Bob. Bob decrypts the ciphertext message with the shared private key, allowing him to recover the plaintext message sent by Alice. 
```

Comment:
lets point out that its a single private key, not a key pair

Reply:
Done